content/faq.md

```diff
@@ -101,6 +101,12 @@ Publishers of individual default UNLs set their own policies for when to add or
 For recommendations and best practices, see [Run `rippled` as a Validator](run-rippled-as-a-validator.html).
 
 
+#### If the dUNL has the most influence on the network, then is the XRPL centralized?
+Validators can choose to not use the dUNL, or any widely-used UNL for that matter. Anyone can create a new UNL at any time.
+
+There can be other UNLs in use on the network. The dUNL is the most widely-used UNL, but validators can choose others.
+
+
```

Comment:
I think we should mention something about the possibility of forking when nodes use different UNLs